statistics-dashboard-common/src/main/java/statistics/dashboard/common/view/ResultListFilters.java

```diff
@@ -0,0 +1,24 @@
+package statistics.dashboard.common.view;
+
+import java.util.List;
+
+/**
+ * A wrapper class that saves all the breakdowns results
+ */
+
+public class ResultListFilters {
+
+  private List<BreakdownResult> allBreakdowns;
+
+  public ResultListFilters(List<BreakdownResult> allBreakdowns) {
+    this.allBreakdowns = allBreakdowns;
+  }
+
+  public List<BreakdownResult> getAllBreakdowns() {
+    return allBreakdowns;
```

Comment:
This goes for all request and response classes. Wherever you get a list, it is good practice to make it immutable (so that the caller cannot mess with the list inside this object). E.g. `Collections.immutableList()`.

Reply:
Done